<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>491</width>
      <height>387</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_HowManyProfile">
       <property name="text">
        <string>How many profile will be created to simulate ? </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_student">
       <property name="text">
        <string>Number of student:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_SETC">
           <property name="text">
            <string>MSc SETC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_nbStudent_MscSETC">
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_amount1">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_amount_MscSETC">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_Energy">
           <property name="text">
            <string>MSc Energy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_nbStudent_MscSEnergy">
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_amount2">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_amount_MscEnergy">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_phdAPS">
           <property name="text">
            <string>Phd Aerospace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_nbStudent_PhdAPS">
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_amount3">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_amount_PhdAerospace">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_researcher">
       <property name="text">
        <string>Number of researcher:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_Defense">
           <property name="text">
            <string>Defense and security</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_nbResearcher_Defense">
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_amount4">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_amount_Defense">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_Aerospace">
           <property name="text">
            <string>Aerospace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_nbResearcher_Aerospace">
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_amount5">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_amount_AeroResearch">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_Environment">
           <property name="text">
            <string>Environment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_nbResearcher_Environment">
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_amount6">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_amount_Env">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_blank_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_IT">
         <property name="text">
          <string>Number of IT support profiles:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_nbIT">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_blank_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_howManyWeek">
         <property name="text">
          <string>How many week will last the simulation ? </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_nbWeek">
           <property name="text">
            <string>Number of week (1 to 53): </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_nbWeek">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>53</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset parameters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_param">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Informations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum parameters</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
